WordPress/WordPressTest/CoreDataStackHelper.swift

```diff
@@ -0,0 +1,49 @@
+import Combine
+import XCTest
+
+@testable import WordPress
+
+extension ContextManagerMock {
+    /// Override `ContextManager.shared` with the mock instance, and un-override it when
+    /// given `testCase` finishes.
+    ///
+    /// This method should be called during test run, typically from `XCTestCase` subclass's
+    /// `setUp` method or test methods.
+    ///
+    /// - Parameter testCase: The test case to wait for.
+    @objc func useAsSharedInstanceUntilTestFinished(_ testCase: XCTestCase) {
```

Comment:
Method naming nitpick: to better follow Swift API conventions, given we now have to pass the test as parameter, I'd suggest to go with something like: `useAsSharedInstance(untilTestFinished testCase: XCTestCase)`